CovidPolicy.md

```diff
@@ -0,0 +1,43 @@
+1.     [**Vaccine Expectation**](#vaccine-expectation)
+1.     [**Negative COVID-19 Test Expectation**](#negative-covid-19-test-expectation)
+
+### Outdoor Event Guidelines
+
+In line with San Francisco guidelines, we prioritize the well-being of our attendees at this outdoor event. While masks and vaccinations are not currently mandated, we reserve the right to change this at any time to comply with San Francisco guidelines. Refunds will be offered for attendees who are unable to comply with updated guidelines. At this time, personal choices regarding their usage are respected. We encourage everyone to uphold good hygiene practices, maintain social awareness, and consider vaccination for the safety of all. Should you feel unwell or exhibit symptoms, we kindly request staying home. Your health and enjoyment are paramount, and we're committed to fostering a secure environment throughout the event. We will refund your ticket if you are unwell.
+
+We will have masks available at the conference for anyone who needs them. It is also expected that requests for personal space will be respected by attendees and are covered under the code of conduct. 
+
+While we follow San Francisco's guidance for this outdoor events and don't require masks or proof of vaccination, we respect individual choices. Please prioritize hygiene, personal space, and your health. Stay home if unwell. Your safety matters most as we enjoy the event together.
+
+**At a minimum, we will be following all mandated venue, municipality, and province guidelines. This list, and all protocols, are subject to change if the COVID-19 situation changes.**
+
+### Vaccine Expectation
+
+If you are asked to show proof of vaccination to attend in person, you will need to show that you are ‘fully vaccinated’ against the COVID-19 virus. We expect that you:
+
+*     have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
+*     that your most recent dose was at least fourteen days before the conference start
+*     that your most recent dose or booster was no more than twelve months before the conference start
+
+The following are acceptable forms of proof of full vaccination:
+
+*     Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
+*     Physical (or digital photocopy of) COVID-19 vaccination card, record, or certificate issued by a government agency, public health agency, or other authorized vaccine provider (a photo will also suffice).
+
+The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
+
+### Negative COVID-19 Test Expectation
```

Comment:
If we are not requiring vaccination, having something in lieu of that seems also weird. It would be nice if we could commit to, and mention, that we will have some testing kits for people to self-administer before they enter the venue on a voluntary basis.